src/main/java/com/minecolonies/core/generation/ItemNbtCalculator.java

```diff
@@ -57,7 +74,18 @@ public String getName()
     @Override
     public CompletableFuture<?> run(@NotNull final CachedOutput cache)
     {
-        return lookupProvider.thenCompose(provider ->
+        final ResourceManager serverResources = loadServerData();
+
+        // Force loading some tags, since the creative tabs don't enumerate properly without them
+        return lookupProvider

```

Comment:
if it is a loading order thing, why dont we just use creative tabs later?

Reply:
because this is the event to do handling and synching.